src/utils/screens.ts

```diff
@@ -15,6 +15,7 @@ export const SCREENS = {
   RESET_PASSWORD: '/reset-password',
   EDIT_PROFILE: '/edit-profile',
   EDIT_MONITORING: '/edit-monitoring',
+  EDIT_MONITORING_EDITMODE: '/edit-monitoring?edit=true',
```

Comment:
Não precisa colocar os query params aqui:

```suggestion
  EDIT_MONITORING_EDITMODE: '/edit-monitoring,
```